src/main/java/mate/academy/Main.java

```diff
@@ -2,15 +2,16 @@
 
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Main {
+    private static final int NUMBER_OF_THREADS = 5;
+
     public static void main(String[] args) {
-        // Feel free to play with AsyncRequestProcessor in this main method if you want
-        ExecutorService executor = null; // Provide implementation that fits your needs
+        ExecutorService executor = Executors.newFixedThreadPool(NUMBER_OF_THREADS);
```

Comment:
Consider using try-with-resources or a shutdown hook to ensure that the ExecutorService is properly shutdown in case of an exception or early return from the method.